include/z64actor.h

```diff
@@ -501,9 +501,15 @@ typedef enum DoorLockType {
 #define ACTOR_FLAG_800           (1 << 11)
 
 // Actor will not shake when a quake occurs
-#define ACTOR_FLAG_IGNORE_QUAKE  (1 << 12)
-// 
-#define ACTOR_FLAG_2000          (1 << 13)
+#define ACTOR_FLAG_IGNORE_QUAKE (1 << 12)
+
+// The hookshot is currently attached to this actor.
+// The behavior that occurs after attachment is determined by `ACTOR_FLAG_200` and `ACTOR_FLAG_400`.
+// If neither of those flags are set attachment cannot occur, and the hookshot will simply act as a damage source.
+//
+// This flag is also reused to indicate that an actor is attached to the Zora boomerang.
+// This only has an effect for Gold Skulltula Tokens (EN_SI) which has overlapping behavior for hookshot and boomerang
```

Comment:
Extremely tiny thing but I just pushed an change in oot to add a period to the end of the last sentence